.github/workflows/ci.yml

```diff
@@ -44,10 +44,10 @@ jobs:
           - distro: rockylinux9
             playbook: converge.yml
             experimental: false
-          - distro: ubuntu2404
+          - distro: ubuntu2204
```

Comment:
This is reverting testing back to older releases of Ubuntu and Debian FYI - please make sure these are both kept at 24.04 and 12.

Reply:
:+1: Will fix. Would you accept a PR on testing multiple Ubuntu versions?
It would be nice to know whether the PR works on older releases as well :)